rwt_image_view/launch/example.launch

```diff
@@ -1,6 +1,10 @@
 <launch>
-  <node pkg="roswww" type="webserver.py" name="roswww">
-  </node>
+  <arg name="launch_roswww" default="true" />
+  <include file="$(find roswww)/launch/roswww.launch"
+           if="$(arg launch_roswww)">
+    <arg name="port" value="8000" />
```

Comment:
Also wonder if the default port in [roswww.launch is 8085](https://github.com/tork-a/roswww/blob/32f58a1911c25edaa12960e035f0eacb1d215da0/launch/roswww.launch#L3) is not suitable for your usage?